### PR TITLE
Regression Tests: Buttons and checkbox/choicegroup regression tests

### PR DESCRIPTION
**Pull request by gitjain**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/CommandButtonPage.visualtest.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { CommandButton } from './index';
+
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+export default class CommandButtonVPage extends React.Component<any, any> {
+  public render() {
+    return <div style={ { backgroundColor: 'white' } }>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Button/CommandButtonPage.visualtest.tsx]: Command buttons usually have icons. Add icon to test it too.

File: packages/office-ui-fabric-react/src/components/Button/CompoundButtonPage.visualtest.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { CompoundButton } from './index';
+
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+export default class CompoundButtonVPage extends React.Component<any, any> {
+  public render() {
+    return <div >
+      <CompoundButton id='CompoundButton'> Compound Button </CompoundButton>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Button/CompoundButtonPage.visualtest.tsx]: Compound button has a description prop. Add that in too.

File: packages/office-ui-fabric-react/src/components/Button/DefaultButtonPage.visualtest.tsx [hunk 3]
```diff
@@ -3,11 +3,11 @@ import { IconName } from '../../Icon';
 /* tslint:disable:no-unused-variable */
 import * as React from 'react';
 /* tslint:enable:no-unused-variable */
-export default class ButtonVPage extends React.Component<any, any> {
+export default class DefaultButtonVPage extends React.Component<any, any> {
   public render() {
     let iconName: IconName = 'Snow';
     return <div >
-      <DefaultButton id='DefaultButton'> I'm a button! </DefaultButton>
+      <DefaultButton id='DefaultButton'> Default Button </DefaultButton>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Button/DefaultButtonPage.visualtest.tsx]: Can have icon, go ahead and test it with the icon

File: packages/office-ui-fabric-react/src/components/Button/PrimaryButtonPage.visualtest.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { PrimaryButton } from './index';
+
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+export default class PrimaryButtonVPage extends React.Component<any, any> {
+  public render() {
+    return <div >
+      <PrimaryButton id='PrimaryButton' > Primary Button </PrimaryButton>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Button/PrimaryButtonPage.visualtest.tsx]: Can have icon, go ahead and test it with the icon

**Comment by micahgodbolt** [comment]
Missing testing contextual buttons. They aren't a specific variant, but we want to make sure to test how they look.

**Comment by micahgodbolt** [comment]
Also, we aren't testing the look of the disabled state. I'd suggest each page have the button and a disabled button with a different ID. Then you can test it, with hover and click (to verify they don't change).

File: packages/office-ui-fabric-react/gulpfile.js [hunk 5]
```diff
@@ -20,7 +20,7 @@ let {
 } = buildConfig;
 
 let visualTestClean = build.subTask('visualTestClean', (gulp, options, done) => {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/gulpfile.js]: hmm this one seems unnecessary. There is a clean config where we can provide the match. We shouldn't need to tdo this.

**Comment by gitjain** [comment]
Choice group pngs shows the similar error mentioned in  "css issue in ChoiceGroup in IE11 #1448"
Though the codepen is working fine. Let me know if I am missing something.
http://codepen.io/reenajain/pen/gmVEVd

File: packages/office-ui-fabric-react/gulpfile.js (hunk 5 above)
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/gulpfile.js]: This is just a temporary cleanup. We'll move this over to the clean config in the future.

**Pull request merged**